db/migrate/20161114111520_add_project_id_to_versions.rb

```diff
@@ -1,4 +1,4 @@
-class AddProjectIdToVersions < ActiveRecord::Migration[5.0]
+class AddProjectIdToVersions < ActiveRecord::Migration[5.1]
```

Comment:
We changed the ported migrations to 5.1, as we did with the rest of migrations long ago. But this one was still in 5.0. Ported too.